src/sentry/static/sentry/app/components/quickTrace/index.tsx

```diff
@@ -303,7 +313,7 @@ function EventNodeSelector({
         title={
           <StyledEventNode text={text} pad={pad} hoverText={hoverText} type={type} />
         }
-        anchorRight
+        anchorRight={anchorRight}
```

Comment:
I think this needs to be passed to the `DropdownNodeItem` as well so it can expand towards the right instead of left.